flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/aggregate/window/buffers/RecordsWindowBuffer.java

```diff
@@ -164,12 +177,24 @@ public WindowBuffer create(
             return new RecordsWindowBuffer(
```

Comment:
How often is this method called during runtime? Do we need to cache calls to ExecutionConfig to avoid string-parsing and conversion overhead?

Reply:
This object is constructed when the operator is opened, so I don't think that parse results should be cached.